pkg/controller/controller.go

```diff
@@ -81,9 +82,17 @@ func NewManager(cfg *rest.Config, options ctrl.Options) (ctrl.Manager, error) {
 		return nil, lerr
 	}
 	podSelector = podSelector.Add(*podLabels)
+
+	taskrunSelector := labels.NewSelector()
+	trLabels, lerr := labels.NewRequirement(TaskLabel, selection.In, []string{"buildah-remote-oci-ta", "clean-shared-host", "provision-shared-host", "update-host"})
```

Comment:
humm, I have a concern about this change. First, there are more task to watch than buildah-remote-oci-ta, there is the regular one and all the various flavors of buildah with larger resources.

The concern is everytime a task will be added, MPC will not react until we update this.

Reply:
I agree, this proposal far from ideal. But unfortunately we have no other options to filter the TaskRun-s multiplicity, other than by labels or some common fields(like `metadata.Name` and `metadata.Namespace`).
 
As far as i was able to understood from docs, caching of objects does not affects ability of their processing by controller (and this is proofed by the fact that most konflux controlles does not use caching at all), it is just means they're always will be read directly from etcd on `Get` operations which is increases the api load and slowing down the reconcillation speed.